test/logic_array_test.dart

```diff
@@ -477,6 +477,18 @@ class IndexBitOfArrayModule extends Module {
   }
 }
 
+class AssignSubsetModule extends Module {
+  AssignSubsetModule(List<Logic> updatedSubset, {int? start}) {
+    final o = LogicArray([10], 3, name: 'logicArray');
+    o <= Const(LogicValue.ofString('1').replicate(10 * 3));
```

Comment:
we need to leave at least the signals not included in updatedSubset disconnected